cidc_api/models/files/details.py

```diff
@@ -12,62 +12,118 @@ class FileDetails(NamedTuple):
         "analysis",
         "comma-separated two-column table with cell counts for each assigned cell type",
         "A plain-text, comma-separated table with a numbered index column, the 'CellSubset' as the called cell type, and 'N', the number of cells of that type seen in the sample.",
     ),
-    "/cytof_analysis/cell_counts_compartment.csv": FileDetails(
+    "/cytof_10021_analysis/cell_counts_compartment.csv": FileDetails(
         "analysis",
         "comma-separated two-column table with cell counts for each broad compartment assigned",
         "A plain-text, comma-separated table with a numbered index column, the 'CellSubset' as the broad compartment of the called cell types, and 'N', the number of cells within that compartment seen in the sample.",
     ),
-    "/cytof_analysis/cell_counts_profiling.csv": FileDetails(
+    "/cytof_10021_analysis/cell_counts_profiling.csv": FileDetails(
         "analysis",
         "comma-separated two-column table with cell counts for each profiled subset of all assigned cell types",
         "A plain-text, comma-separated table with a numbered index column, the 'CellSubset' as the profiled subset of the assigned cell types, and 'N', the number of cells within that profiled subset seen in the sample.",
     ),
-    "/cytof_analysis/assignment.csv": FileDetails(
+    "/cytof_10021_analysis/assignment.csv": FileDetails(
         "analysis",
         "comma-separated table of marker expression for each assigned cell type",
         "A plain-text, comma-separated table with a column for each assigned cell type, where rows are the signal on each channel for every cell type assigned.",
     ),
-    "/cytof_analysis/compartment.csv": FileDetails(
+    "/cytof_10021_analysis/compartment.csv": FileDetails(
         "analysis",
         "comma-separated table of marker expression for each broad compartment assigned",
         "A plain-text, comma-separated table with a column for each broad compartment of the called cell types, where rows are the signal on each channel for every compartment.",
     ),
-    "/cytof_analysis/profiling.csv": FileDetails(
+    "/cytof_10021_analysis/profiling.csv": FileDetails(
         "analysis",
         "comma-separated table of marker expression for each profiled subset of all assigned cell types",
         "A plain-text, comma-separated table with a column for each profiled subset of all assigned cell types, where rows are the signal on each channel for every profiled subset.",
     ),
-    "/cytof_analysis/source.fcs": FileDetails(
+    "/cytof_10021_analysis/source.fcs": FileDetails(
         "source",
         "fcs data used as the input for this analysis",
         "The analysis-ready FCS file used as the input for this analysis. After normalization, debarcoding, and removal of Veri-Cells and other non-specimen cells.",
     ),  # schemas/assays/components/cytof/cytof_analysis.json#properties/fcs_file
+    # CyTOF E4412
+    "/cytof_e4412/spike_in.fcs": FileDetails(
```

Comment:
I wonder if it's worth factoring out identical `FileDetails` definitions across cytof variants into shared variables (so we that we don't need to edit descriptions in two places, e.g.).

Reply:
yeah, that's not too bad. in next commit